lib/message_express/bus/rd_kafka.rb

```diff
@@ -0,0 +1,37 @@
+require 'message_express/bus/abstract_bus'
+require 'message_express/message'
+
+module MessageExpress
+  module Bus
+    class RdKafka < AbstractBus
+      attr_reader :kafka
+
+      def initialize(kafka_client, topic: nil)
+        @kafka = kafka_client
+        @topic = topic || 'messages'
+      end
+
+      def publish(message)
+        producer_params = {
+          topic: @topic,
+          payload: message,
+          key: "#{@topic}-#{rand(10**10)}"
```

Comment:
Why do you need to define the key? The gem is not taking care of it?

Reply:
No it's not in this case. At first I'd thought that also... but errors arose, and I had to add it.